src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -1,17 +1,21 @@
 package core.basesyntax;
 
-import java.util.Collections;
 import java.util.List;
+import java.util.NoSuchElementException;
 import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 public class JavaStreamApi {
 
     /**
      * <p>1. Дано: List of Integer numbers.
      * Вернуть сумму нечетных числел или 0, если таких несуществует</p>
-     **/
+     * */
     public Integer oddSum(List<Integer> numbers) {
-        return null;
+        return numbers.stream()
+                .filter(x -> x % 2 != 0)
+                .reduce(0,(first, second) -> first + second);
```

Comment:
```suggestion
                .reduce(0, Integer::sum);
```
твій варіант теж вірний, це для практики і кращого розуміння :)